libs/core/src/components/sage-avatar/sage-avatar.tsx

```diff
@@ -0,0 +1,77 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-avatar',
+  styleUrl: 'sage-avatar.scss',
+  shadow: true,
+})
+export class SageAvatar {
+
+  /**
+   * The alt for a custom user image.
+   * @defaultValue null
+   */
+  @Prop() alt?: string | null;
+
+  /**
+   * Determines whether the badge is visible or not
+   * @defaultValue false
+   */
+  @Prop() badge? = false;
+
+  /**
+   * The src for a custom user image.
+   * @defaultValue null
+   */
+  @Prop() image?: string | null;
+
+  /**
+   * Preset sizes for the avatar. If a custom size is desired,
+   * use the `--size` custom property instead.
+   */
+  @Prop() size?: 
+  | 'xl'
+  | 'lg'
+  | 'md' 
+  | 'sm' 
+  | 'xs';
+
+  /**
+   * Determines the variant of avatar. Changes appearance accordingly.
+   * @defaultValue customer
+   */
+  @Prop({ reflect: true }) variant?: 'customer' | 'admin' = 'customer'
+
+  private renderBadge = () => (
+    this.badge
+      && <sage-icon class="sage-avatar__badge" name="check-circle-filled" size="normal"></sage-icon>
```

Comment:
you could pass your 33.53% here to the `size` prop, which would remove the need for the style in the `sage-avatar.scss`. 

Also named values for `size` are below...This may be why you were having issues with appropriate sizing. Default is `regular` if you don't pass `size`
```
'small'   // 12px
    | 'regular'  // 16px
    | 'medium'  // 20px
    | 'large'   // 24px
    | 'auto'
```